packages/mdl-icon-toggle/README.md

```diff
@@ -0,0 +1,199 @@
+# MDL Icon Toggle
+
+MDL Icon Toggle provides a material design icon toggle button. It is fully accessible, and is
```

Comment:
That's a great question. It shouldn't according to the [W3C spec](https://www.w3.org/TR/wai-aria/roles#textalternativecomputation):

Algorithm rule `2A`:

> If aria-labelledby is empty or undefined, the aria-label attribute, which defines an explicit text string, is used...

Algorithm rule `2C`:

> Otherwise, if the attributes checked in rules A and B didn't provide results, text is collected from descendant content if the current element's role allows "Name From: contents."